NDTensors/test/runtests.jl

```diff
@@ -18,6 +18,10 @@ using SafeTestsets: @safetestset
     using NDTensors: NDTensors
     include(joinpath(pkgdir(NDTensors), "ext", "examples", "NDTensorCUDA.jl"))
   end
+  if "rocm" in ARGS || "all" in ARGS
+    using NDTensors: NDTensors
```

Comment:
It seems better to just have one call to `using NDTensors: NDTensors` at the top of the file.

Reply:
Okay I have updated this!